src/main/java/core/basesyntax/service/impl/TransactionParserImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.TransactionParser;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TransactionParserImpl implements TransactionParser {
+    private static final String SPLITERATOR = ",";
+    private static final int OPERATION_TYPE_INDEX = 0;
+    private static final int FRUIT_TYPE_INDEX = 1;
+    private static final int FRUITS_QUANTITY_INDEX = 2;
+
+    @Override
+    public List<FruitTransaction> parseData(List<String> dataFromFile) {
+        List<FruitTransaction> fruitTransactions = new ArrayList<>();
+        for (int i = 1; i < dataFromFile.size(); i++) {
```

Comment:
```suggestion
        for (int i = OFFSET; i < dataFromFile.size(); i++) {
```
let's make it visible in code that we are skipping the first line